doc/source/dev/contributor/adding_notebooks.rst

```diff
@@ -0,0 +1,146 @@
+  can use double asterisks: ``**bold text**``. To make text monospace/formatted
+  as code, you can use *single* backticks: ```code```.
+- **Images:** If your notebook contains static images, you can include them in
+  the MyST Markdown file by using the following syntax:
+  ``![alt text](path/to/image.png)``. Images are usually stored in the same
+  folder as the MyST Markdown file, but you can also use relative paths to
+  reference images in other folders. Note that outputs of executed cells should
+  not be included in version control, as they will be automatically generated
+  when the notebook is executed during the SciPy documentation build.
+- **Linking to the MyST markdown pages:** MyST markdown supports adding link
+  anchors, which can be used to link to specific pages or document sections from
+  other documents. To add an anchor to a page, add ``(anchor_name)=`` to the
+  desired place in the document to be referenced. From other markdown pages, you
+  can link to this anchor using the following syntax: ``{ref}`anchor_name```.
+  From other reST pages, you can link to this anchor using the following syntax:
+  ``:ref:`anchor_name```.
+
+Making the notebook interactive
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to enable interactivity for your notebook when it is rendered in
+the SciPy documentation website, you need to add the following snippet at the
+beginning of the markdown file:
+
+.. code-block:: markdown
+
+    ```{eval-rst}
+    .. notebooklite:: <filename>.md
+       :new_tab: True
+    ```
+
+where ``<filename>.md`` should be replaced with the name of the file you are
+working with. This will create a button that allows users to open the notebook
+in a new tab and interact with it. See
+`Bartlett's test for equal variances <https://scipy.github.io/devdocs/tutorial/stats/hypothesis_bartlett.html>`__
+(and `its source <https://github.com/scipy/scipy/blob/main/doc/source/tutorial/stats/hypothesis_bartlett.md?plain=1>`__)
+for an example.
+
+.. note::
+
+    For simplicity, we don't want certain notebook cells to be visible in the
+    rendered documentation. We can hide them by adding ``"jupyterlite_sphinx_strip"``
+    to the tags section of the corresponding cell metadata. To do that in the
+    markdown file, add the following snippet right before cell you want to hide:
+
+    .. code-block:: markdown
+
+        +++ {"tags": ["jupyterlite_sphinx_strip"]}
+
+Visit the `NotebookLite directive documentation <https://jupyterlite-sphinx.readthedocs.io/en/latest/directives/notebooklite.html>`__
+for more details and options.
+
+Opening MyST Markdown files in the Jupyter Notebook application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have the ``jupytext`` tool installed, you can open MyST Markdown ``.md``
+files in the Jupyter Notebook application and execute them, just as you would
+with a ``.ipynb`` file.
+
+Converting ``.rst`` files to MyST Markdown
```

Comment:
@agriyakhetarpal I think you mentioned `rst-to-myst`, initially? Would it be OK to add a note about it at the end of the "Links, formatting and images" section, which is mostly about converting to markdown, instead of having its own section?